### ISSUE TITLE
Remove black ring on focus when copying private link

### ISSUE THREAD

**Issue by CarinaWolli** [calcom/cal.com#4797]
### Issue Summary

When copying the private link in the advanced event-type settings it currently looks like this: 
<img width="893" alt="Screenshot 2022-10-03 at 13 23 40" src="https://user-images.githubusercontent.com/30310907/193565757-093e7ecc-ab51-4b38-9f7b-7b4ce2ff931a.png">

The black ring around the copy icon should not be shown.

**Comment by MuriloAmarals** [comment]
Hey @CarinaWolli I'll send you a print of how it turned out, do you want me to take the ring off just the button?

**Comment by MuriloAmarals** [comment]
Take a look at this!

https://www.loom.com/share/2b7458501b63435494187d7f7b5fe3d9

**Comment by CarinaWolli** [comment]
> Take a look at this! https://www.loom.com/share/2b7458501b63435494187d7f7b5fe3d9

@MuriloAmarals Looks already way better! Would it be possible to have the ring only around the gray area when clicking the button?

**Comment by MuriloAmarals** [comment]
> > Take a look at this! https://www.loom.com/share/2b7458501b63435494187d7f7b5fe3d9
> 
> @MuriloAmarals Looks already way better! Would it be possible to have the ring only around the gray area when clicking the button?

Let me try!!

**Comment by MuriloAmarals** [comment]
@CarinaWolli Is good?
https://www.loom.com/share/4e6c03ffea5c415b9a8cfcd807041891

**Comment by CarinaWolli** [comment]
> @CarinaWolli Is good?
https://www.loom.com/share/4e6c03ffea5c415b9a8cfcd807041891

@MuriloAmarals What exactly changed? I don't see anything changed from the first video

**Comment by MuriloAmarals** [comment]
> > @CarinaWolli Is good?
> > https://www.loom.com/share/4e6c03ffea5c415b9a8cfcd807041891
> 
> @MuriloAmarals What exactly changed? I don't see anything changed from the first video

In the first video you can't see it because of the quality, but there is space between the ring and the border. In the second video I left it together, it got better.

**Comment by MuriloAmarals** [comment]
But I solved the issue bug, right?

**Comment by CarinaWolli** [comment]
> But I solved the issue bug, right?

Yes, looks good so far! Please just create a PR and link this issue, then I can review it :)

**Comment by MuriloAmarals** [comment]
Hey @CarinaWolli look what I got.
https://www.loom.com/share/48409b9905114c67bb68efbfe40d59bc

**Comment by MuriloAmarals** [comment]
> Hey @CarinaWolli look what I got. https://www.loom.com/share/48409b9905114c67bb68efbfe40d59bc

But in the PR I sent I didn't leave it that way, because I couldn't leave the ring just around the button.
